src/main/java/io/dashboardhub/pipelinedashboard/config/AuditingDateTimeProvider.java

```diff
@@ -0,0 +1,20 @@
+package io.dashboardhub.pipelinedashboard.config;
+
+import org.springframework.data.auditing.DateTimeProvider;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
+public class AuditingDateTimeProvider implements DateTimeProvider {
```

Comment:
This file ends in **Provider**, should this not go into a **Provider** directory?